components/NavigationDrawer/src/MDCBottomDrawerPresentationController.h

```diff
@@ -87,4 +87,9 @@
  */
 @property(nonatomic, strong, nullable) UIColor *topHandleColor;
 
+/**
+ Whether the content/drawer reaches the full screen of the device.
+ */
+@property(nonatomic, readonly) BOOL contentReachesFullscreen;
```

Comment:
Would you please add a test to the PresentationController to confirm that it returns the value supplied by its `bottomDrawerContainerViewController`?